src/unistd.rs

```diff
@@ -1398,6 +1398,30 @@ pub fn setgid(gid: Gid) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Set the user identity used for filesystem checks.
+/// On both success and failure, this call returns the previous filesystem user
+/// ID of the caller.
+///
+/// See also [setfsuid(2)](http://man7.org/linux/man-pages/man2/setfsuid.2.html)
+#[cfg(target_os = "linux")]
+#[inline]
```

Comment:
No need for `#[inline]`

Reply:
I removed the attribute (fyi I added it just because it is used for all the other equivalent functions in the same module).